pkg/graveler/graveler.go

```diff
@@ -297,6 +299,8 @@ type CommitRecord struct {
 type Branch struct {
 	CommitID     CommitID
 	StagingToken StagingToken
+	// SealedTokens - Staging tokens are appended to the back so that iteration of tokens will be from oldest to newest
```

Comment:
We're writing once to the sealed tokens from a single location and reading many times. It might be worth considering to make the one time operation more complicated than the many times operations.